restli-tools/src/test/java/com/linkedin/restli/tools/snapshot/circular/CircularResource.java

```diff
@@ -31,7 +31,7 @@
 @RestLiActions(name="circular")
 public class CircularResource
 {
-  @Action(name="test")
+  @Action(name="test", readOnly = true)
```

Comment:
I am curious about how this hint will be consumed? By reading the resourceClass Java class?

Reply:
Yes I updated the PR to add this to the IDL and Snapshot.